### PR TITLE
Speaker with new layout

### PR DESCRIPTION
**Pull request by drublic**

### CODE REVIEW

**Comment by asciidisco** [comment]
Looks way better than before. Well done. Have you tried how it looks with 5 speakers or any idea how we can highlight the MC?

**Review by asciidisco — APPROVED** [review]

File: lib/views/pages/speakers.html [hunk 1]
```diff
@@ -2,88 +2,33 @@
...
-        <ul class="links">
-            <li><a href="http://www.the-haystack.com/">web</a></li>
-            <li><a href="https://twitter.com/stephenhay">twitter</a></li>
-        </ul>
-    </div>
-</figure>
+<div class="clearfix">
+
+    <figure id="janabeck" class="speaker">
+        <img src="/img/speaker/jana.jpg" alt="" class="speaker_image" />
```
Comment (anselmh) [inline lib/views/pages/speakers.html]: Alt text?

File: lib/views/pages/speakers.html [hunk 2]
```diff
@@ -2,88 +2,33 @@
...
+            <p class="description">Jana is a front-end engineer who builds data visualization apps &amp; tools with D3, React, and Redux.</p>
+            <ul class="links--fancy">
+                <li><a href="http://janabeck.com/">web</a></li>
+                <li><a href="https://twitter.com/iPancreas">twitter</a></li>
+            </ul>
+        </div>
+    </figure>
+
+    <figure id="amberwilson" class="speaker">
+        <img src="/img/speaker/amber.jpg" alt="" class="speaker_image" />
```
Comment (anselmh) [inline lib/views/pages/speakers.html]: alt?

File: lib/views/pages/speakers.html (hunk 1 above)
Comment (drublic) [reply lib/views/pages/speakers.html]: Both fixed.

**Comment by drublic** [comment]
> Looks way better than before. Well done. Have you tried how it looks with 5 speakers or any idea how we can highlight the MC?

5 + 1 fits good on 2col and 3col grid. MC should be highlighted. Maybe with an outline or lighter background. What do you think?

**Review by anselmh — APPROVED** [review]

**Pull request merged**